spec/asyncapi.md

```diff
@@ -524,6 +532,42 @@ servers:
 ```
 
 
+#### <a name="serverTraitObject"></a>Server Trait Object
+
+Describes a trait that MAY be applied to a [Server Object](#serverObject). This object MAY contain any property from the [Server Object](#serverObject), except `url`, `protocol` and `protocolVersion`.
+
+##### Fixed Fields
+
+Field Name | Type | Description
+---|:---:|---
+<a name="serverTraitObjectDescription"></a>description | `string` | An optional string describing the host designated by the URL. [CommonMark syntax](https://spec.commonmark.org/) MAY be used for rich text representation.
+<a name="serverTraitObjectVariables"></a>variables | Map[`string`, [Server Variable Object](#serverVariableObject) \| [Reference Object](#referenceObject)]] | A map between a variable name and its value.  The value is used for substitution in the server's URL template.
+<a name="serverTraitObjectSecurity"></a>security | [[Security Requirement Object](#securityRequirementObject)] | A declaration of which security mechanisms can be used with this server. The list of values includes alternative security requirement objects that can be used. Only one of the security requirement objects need to be satisfied to authorize a connection or operation.
+<a name="serverTraitObjectTags"></a>tags | [Tags Object](#tagsObject) | A list of tags for logical grouping and categorization of servers.
+<a name="serverTraitObjectBindings"></a>bindings | [Server Bindings Object](#serverBindingsObject) \| [Reference Object](#referenceObject) | A map where the keys describe the name of the protocol and the values describe protocol-specific definitions for the server.
+
+This object MAY be extended with [Specification Extensions](#specificationExtensions).
+
+##### Server Trait Object Example
+
+```json
+{
+  "tags": [
+    { 
+      "name": "env:production",
+      "description": "This environment is the live environment available for final users"
+    }
+  ]
+}
+```
+
+```yaml
+tags:
+  - name: "env:production"
+    description: "This environment is the live environment available for final users"
+```
```

Comment:
Can you come up with a more complete example? For instance, `security` and `variables` would be super useful in many cases.

Reply:
I made such simple examples of what operations and message traits have :trollface: I will extend it.